File: 2017.1/2017-04-25/Services/app/src/main/java/br/ufpe/cin/if1001/services/MusicPlayerNoBindingActivity.java
package br.ufpe.cin.if1001.services;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MusicPlayerNoBindingActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_music_player_no_binding);

        // Intent usado para iniciar Player de Musica
        final Intent musicServiceIntent = new Intent(this, MusicPlayerNoBindingService.class);

        final Button startButton = (Button) findViewById(R.id.btn_StartService);
        startButton.setOnClickListener(new View.OnClickListener() {
            public void onClick(View src) {
                // inicia o service usando o intent
                startService(musicServiceIntent);
            }
        });

        final Button stopButton = (Button) findViewById(R.id.btn_StopService);
        stopButton.setOnClickListener(new View.OnClickListener() {
            public void onClick(View src) {
                // para o service usando o mesmo intent
                stopService(musicServiceIntent);
            }
        });
    }
}

File: 2018.1/2018-04-04/Services/app/src/main/java/br/ufpe/cin/if710/services/MusicPlayerNoBindingActivity.kt
package br.ufpe.cin.if710.services

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button

class MusicPlayerNoBindingActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_music_player_no_binding)

        val serviceIntent = Intent(this, MusicPlayerNoBindingService::class.java)

        val startButton = findViewById<View>(R.id.btn_StartService) as Button
        val stopButton = findViewById<View>(R.id.btn_StopService) as Button
        startButton.setOnClickListener { startService(serviceIntent) }

        stopButton.setOnClickListener { stopService(serviceIntent) }
    }
}
